### PR TITLE
explorer: Add block rewards to block details page

### PR DESCRIPTION
**Pull request by oJshua**
#### Problem
Explorer does not display block rewards (https://explorer.solana.com/block/51932256). This is available from RPC and displayed in the CLI. 

#### Summary of Changes
- Add block rewards to the block details page

Fixes https://github.com/solana-labs/solana/issues/14045

### CODE REVIEW

**Comment by oJshua** [comment]
blockrewards

```
$ solana block 61904263|grep %

  E1SFkvPjU31xWMcvgnX6vhGvfRvb1zXvHFkqmQNEGZKK        fee        ◎0.002107500     ◎393.940422295          0.000534982%

```

File: explorer/src/components/block/BlockRewardsCard.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+              let percentChange;
+              if (reward.postBalance !== null && reward.postBalance !== 0) {
+                percentChange = (
+                  (Math.abs(reward.lamports) /
+                    (reward.postBalance - reward.lamports)) *
+                  100
+                ).toFixed(9);
+              }
+              return (
+                <tr>
```
Comment (jstarry) [inline explorer/src/components/block/BlockRewardsCard.tsx:39]: You'll want a key here

**Review by jstarry — DISMISSED** [review]
Nice! Are transaction fees the only reward type right now?

**Comment by mvines** [comment]
> Nice! Are transaction fees the only reward type right now?

Rent, Voting and Staking rewards as well.

File: explorer/src/components/block/BlockRewardsCard.tsx (hunk 1 above)
Comment (oJshua) [reply explorer/src/components/block/BlockRewardsCard.tsx:39]: Thanks!

**Comment by jstarry** [comment]
> > Nice! Are transaction fees the only reward type right now?
> 
> Rent, Voting and Staking rewards as well.

Ah, just on epoch boundaries. @oJshua do you mind paginating new epoch blocks? https://explorer.solana.com/block/62208000 I think we should have a page for each epoch as well at some point

**Comment by oJshua** [comment]
> Ah, just on epoch boundaries. @oJshua do you mind paginating new epoch blocks? https://explorer.solana.com/block/62208000 I think we should have a page for each epoch as well at some point

Oh yes, thanks for pointing that out. I'm on it.

**Pull request merged**